duracloud/duracloud
# Set Retrieval Tool default log level to INFO and remove log lines

**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**
* * *

**JIRA Ticket**: https://jira.duraspace.org/browse/DURACLOUD-1210

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)


# How should this be tested?

Compile the retrieval tool and run it. Verify that the complete.log doesn't contain duplicate entries and verify that the log level is INFO.
Repeat using the -Dlog.level=DEBUG command line option.  
Verify that the log level is indeed DEBUG for all org.duracloud.* classes.

@bbranan 
# Additional Notes:
Any additional information that you think . lpful when reviewing this PR.

Example:
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? 
* Could this change impact execution of existing code?

# Interested parties
Tag (@ mention) interested parties or, if unsure, @duracloud/committers
